packages/serverpod_test/lib/src/with_serverpod.dart

```diff
@@ -53,15 +53,18 @@ typedef TestClosure<T> = void Function(
   T endpoints,
 );
 
+const String _defaultIntegrationTestTag = 'integration';
```

Comment:
Suggestion: Export the default tag so that this is extendable.

Reply:
196fb6f